manifests/jail/sasl.pp

```diff
@@ -2,9 +2,15 @@
   $maxretry = 'usedefault',
   $findtime = false,
   $ignoreip = false,
-  $logpath  = '/var/log/mail.log'
```

Comment:
hmm this change removes the possibility for users to override the default value for $logpath. that's a loss to all the ppl who are not using the default MTA for their distro, in case the alternative MTA is logging somewhere else.

maybe it'd be better to have some kind of default value that indicates we should set it to the per-distro default value. (something like 'usedefault' that is used for $maxretry for example)